### ISSUE TITLE
[question] remote computer

### ISSUE THREAD

**Issue by whathappenedman** [casualsnek/cassowary#23]
Can this be used to display a "seamless" from another computer, over rdp?

**Comment by casualsnek** [comment]
Yes it should be able to do that, install the windows component and use it's IP address on linux app. If you get into any issue feel free to open a new issue !
